integrationtests/test_all_examples.py

```diff
@@ -371,7 +404,7 @@ def test_button_editor_simple_demo(self):
             demo.click_counter = 10
             displayed_count = click_counter.inspect(query.DisplayedText())
             self.assertEqual(displayed_count, '10')
-        
```

Comment:
Looks like we can remove `integrationtests/test_all_examples.py` from `setup.cfg` after this change. Then CI will help us keeping this file pep8-clean. Not a request change. But feel free to do as a drive-by.